src/features/rewards/walletWrapper/index.tsx

```diff
@@ -71,7 +92,9 @@ export interface Props {
   grants?: Grant[]
   alert?: AlertWallet | null
   id?: string
-  gradientTop?: string
+  gradientTop?: string,
+  notification?: Notification,
+  onCloseNotification?: () => void
```

Comment:
can be this part of `Notification` interface so that everything about notification is grouped

Reply:
also this needs to change to `onCloseNotification?: (id: string) => void`, so that we know which one we are closing